concept/type/impl/AttributeTypeImpl.java

```diff
@@ -96,14 +98,6 @@ public static AttributeTypeImpl of(GraphManager graphMgr, TypeVertex vertex) {
         }
     }
 
-    @Override
-    public void setAbstract() {
-        if (getInstances().first().isPresent()) {
-            throw exception(TypeDBException.of(TYPE_HAS_INSTANCES_SET_ABSTRACT, getLabel()));
-        }
-        vertex.isAbstract(true);
-    }
-
```

Comment:
the abstract setter for AttributeTypes turns out to be the exact same as for `ThingType` (the generic implementation), so we can remove this. 

It was here from two iterations ago:
1. we originally used to check that when setting attribute type as abstract, it did not have any subtypes
2. we then moved this validation to when we add set the supertype of a new Attribute type: a new attribute type _must_ have an abstract parent

After this second step, this method ends up being the same as the parent implementation but was not removed before